pyaedt/modeler/Primitives.py

```diff
@@ -80,67 +79,59 @@ def __init__(self, type, num_seg=0, num_points=0, arc_angle=0, arc_center=None,
 
 
 class Polyline(Object3d):
-    """Polyline class.
-
-    This class provides methods for creating and manipulating polyline objects within 
-    the AEDT Modeler. Intended usage is for the constructor of this class to be called
-    by the ``Primitives.draw_polyline`` method. The documentation is provided there.
+    """Creates and manipulates a polyline. 
+    
+    The constructor for this class is intended to be called from the 
```

Comment:
```suggestion
    """Creates and manipulates a polyline. 

    The constructor for this class is intended to be called from the 
```
White space suggestion.